libr/core/cmd_type.c

```diff
@@ -41,7 +41,7 @@ static const char *help_msg_t_minus[] = {
 static const char *help_msg_ta[] = {
 	"Usage: ta[...]", "", "",
 	"tas", " <offset>", "List all matching structure offsets",
-	"ta", " <struct member>", "Change immediate to structure offset",
+	"ta", " <struct member> [offset]", "Change immediate to structure offset",
```

Comment:
isnt this [struct.member] ?

Reply:
Probably but did not want to change more than necessary